client/onboarding/index.tsx

```diff
@@ -66,6 +56,7 @@ const OnboardingStepper = () => {
 	);
 };
 
+// TODO: WE CAN PROBABLY JUST ATTACH THIS ON A SERVER LEVEL (URL AND BUSINESS DATA)
```

Comment:
hmm, interesting idea, we may
but it's okay this way I guess

Reply:
Sorry - this is just a TODO note for myself about how we could consider prepopulating the URL and Business name (which we used to ask for in the MOX) when making the server request, because we have this info already. I will remove it or make a new issue for it before merging. 👍

Reply:
no worries, gotcha!